src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/endretutbetaling/EndretUtbetalingAndelServiceTest.kt

```diff
@@ -93,4 +101,49 @@ class EndretUtbetalingAndelServiceTest {
         verify(exactly = 1) { vilkårsvurderingService.hentAktivVilkårsvurderingForBehandling(behandling.id) }
         verify(exactly = 1) { beregningService.oppdaterTilkjentYtelsePåBehandling(behandling, any(), any()) }
     }
+
+    @Test
+    fun `sanityBegrunnelserMedEndringsårsak - skal returnere et map med begrunnelsestyper mappet mot liste av begrunnelser`() {
+        // Arrange
+        val sanityTekster =
+            listOf(
+                SanityBegrunnelse(
+                    NasjonalEllerFellesBegrunnelse.AVSLAG_ENDRINGSPERIODE_ALLEREDE_UTBETALT_ANNEN_FORELDER.sanityApiNavn,
+                    "avslagAlleredeUtbataltAnnenForelderEndringsperiode",
+                    SanityBegrunnelseType.ENDRINGSPERIODE,
+                    Vilkår.values().toList(),
```

Comment:
```suggestion
                    Vilkår.entries.toList(),
```

Ref
''Enum. values()' is recommended to be replaced by 'Enum. entries' since 1.9 '